### ISSUE TITLE
Missing mixin plugin error on startup

### ISSUE THREAD

**Issue by xllifi** [yungnickyoung/YUNGs-Menu-Tweaks#2]
This error shows up when I start the game https://hst.sh/esekuzusim, but the mod works perfectly.

**Comment by SettingDust** [comment]
1.20.1 fabric. Put it here for searchable
```
[20:50:44] [main/ERROR]: Error loading companion plugin class [com.yungnickyoung.minecraft.yungsmenutweaks.YungsMenuTweaksMixinPluginFabric] for mixin config [yungsmenutweaks_fabric.mixins.json]. The plugin may be out of date: ClassNotFoundException:com.yungnickyoung.minecraft.yungsmenutweaks.YungsMenuTweaksMixinPluginFabric
java.lang.ClassNotFoundException: com.yungnickyoung.minecraft.yungsmenutweaks.YungsMenuTweaksMixinPluginFabric
```

**Comment by yungnickyoung** [comment]
Oops, left a line in the mixin config from previous versions that shouldn't be there. Fixed in 1.0.1, thanks for reporting it!
